WorkflowUI/Sources/Hosting/WorkflowHostingController.swift

```diff
@@ -35,6 +35,11 @@ public final class WorkflowHostingController<ScreenType, Output>: WorkflowUIView
         didSet { setNeedsEnvironmentUpdate() }
     }
 
+    /// The currently displayed screen - the most recent rendering from the hosted workflow
+    public var screen: ScreenType {
```

Comment:
what's the use case that need this? i assume one of the various appearance events that contains this type of VC?